google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/MavenBuildDeploymentSource.java

```diff
@@ -40,7 +44,15 @@
     implements AppEngineDeployable {
 
   private final Project project;
-  private final AppEngineEnvironment environment;
+  private AppEngineEnvironment environment;
```

Comment:
I thought we didn't want to serialize environment at all because of that previous freshness issue.